bin/new_ruby_api_service.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'trollop'
+
+opts = Trollop.options do
+  opt :repo, "The repo to update.", :required => true, :type => :strings
+
+  opt :dry_run, "", :default => false
+end
+
+ManageIQ::Release.each_repo(opts[:repo]) do |repo|
+  ManageIQ::Release::UpdateRepoSettings.new(repo.github_repo, opts.slice(:dry_run)).run
+  ManageIQ::Release::UpdateBranchProtection.new(repo.github_repo, opts.slice(:dry_run).merge(:branch => "master")).run
+  ManageIQ::Release::PullRequestBlasterOuter.new(repo, opts.slice(:dry_run).merge(
+    :base    => "master",
+    :head    => "new_ruby_api_service",
```

Comment:
Naming is hard.  It looks like we're pushing your standard docker/travis/codeclimate/test practices for an existing ruby github repository.  I think fryguyify_github_ruby_repo would better describe it.  Are we really creating a new ruby api service?

Reply:
`prepare_ruby_github_repo_for_testing_and_deployment.rb`  I don't know, naming.